packages/wallet/src/PersonalWallet/PersonalWallet.ts

```diff
@@ -145,19 +146,33 @@ export const DEFAULT_LOOK_AHEAD_SEARCH = 20;
 // Ideally we should calculate this based on the activeSlotsCoeff and probability of a single block per epoch.
 const BLOCK_SLOT_GAP_MULTIPLIER = 2.5;
 
-const isOutgoingTx = (input: Cardano.Tx | TxCBOR | OutgoingTx): input is OutgoingTx =>
+const getSerializedTxCBOR = (input: TxCBOR) => TxCBOR.deserialize(input).body;
```

Comment:
```suggestion
const getSerializedTxBody = (input: TxCBOR) => TxCBOR.deserialize(input).body;
```

Reply:
883b14af431d6c40765f8350a71fa6995236fae6 should solve this